carma/launch/ros2_rosbag.launch.py

```diff
@@ -32,7 +32,7 @@ def record_ros2_rosbag(context: LaunchContext, vehicle_config_param_file, rosbag
     vehicle_config_param_file_string = context.perform_substitution(vehicle_config_param_file)
 
     # Initialize string that will contain the regex for topics to exclude from the ROS 2 rosbag
-    exclude_topics_regex = ""
+    exclude_topics = []
```

Comment:
Let's update the comment for this variable since it is no longer a string

Reply:
Done